db/migrate/20200424183342_add_read_only_to_dialogs.rb

```diff
@@ -0,0 +1,5 @@
+class AddReadOnlyToDialogs < ActiveRecord::Migration[5.1]
+  def change
+    add_column :dialogs, :read_only, :boolean
```

Comment:
Just a warning that Rails has a concept of `readonly`, this may cause some confusion.  In some other places I think we've used `system`.